File: StockFlow.Common/Models/SnapshotRate.cs
using System;

namespace StockFlow.Web.Models
{
    public class SnapshotRate
    {
        public int ID { get; set; }
        
        public DateTime Time { get; set; }
        
        public string TimeString
        {
            get
            {
                return Time.ToString("dd.MM.yy");
            }
        }

        public decimal? Open { get; set; }
        public decimal? Close { get; set; }
        public decimal? High { get; set; }
        public decimal? Low { get; set; }
    }
}

File: StockFlow.Node/app/models/snapshotrate.js
'use strict';
module.exports = (sequelize, DataTypes) => {
    var snapshotrate = sequelize.define('snapshotrate', {

        ID: {
            allowNull: false,
            autoIncrement: true,
            primaryKey: true,
            type: DataTypes.INTEGER
        },

        Time: {
            type: DataTypes.DATE,
            allowNull: false
        },

        Open: {
            type: DataTypes.DECIMAL(18, 2),
            allowNull: true
        },

        Close: {
            type: DataTypes.DECIMAL(18, 2),
            allowNull: true
        },

        High: {
            type: DataTypes.DECIMAL(18, 2),
            allowNull: true
        },

        Low: {
            type: DataTypes.DECIMAL(18, 2),
            allowNull: true
        }

    }, {});
    snapshotrate.associate = function (models) {
        snapshotrate.belongsTo(models.snapshot, { foreignKey: 'Snapshot_ID', allowNull: false });
    };
    return snapshotrate;
};